pkg/server/syncHandler.go

```diff
@@ -19,6 +20,11 @@ func (s *ServerConfig) SyncResources(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
 	clusterName := params["id"]
 
+	if !s.Dao.ClusterInDB(clusterName) {
+		klog.Warningf("Warning, couldn't find a Cluster node with name: %s. This means that the sync request came from a managed cluster that hasn’t joined. Rejecting the incoming sync request.", clusterName)
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
```

Comment:
Let's disable this check for now. This was required on RG because every node was connected to the cluster node.  With postgres it's ok if resources are added before the cluster row.

The proxyserver is validating that the cluster is actually registered on ACM.

Reply:
Disabled